zilliqa/src/cfg.rs

```diff
@@ -88,6 +99,11 @@ impl Default for ConsensusConfig {
             minimum_time_left_for_empty_block: Duration::from_millis(3000),
             scilla_address: "http://localhost:3000".to_owned(),
             local_address: "localhost".to_owned(),
+            rewards_per_hour: 32_000_000_000_000u128,
+            blocks_per_hour: 50_000,
+            minimum_stake: 32_000_000_000_000_000_000u128,
```

Comment:
Is there a point in not using the same values as defined in `config.toml`? With an `empty_block_timeout` of 1000ms we would never end up with 50,000 `blocks_per_hour` anyway :-)